### ISSUE TITLE
Creating AdCreative from the example code in the README doesn't work

### ISSUE THREAD

**Issue by kartikluke** [tophatter/facebook-ruby-ads-sdk#12]
```
FacebookAds::AdException: Link title and link description are deprecated in ad creative's call to action. To set title and description for your link ad, use 'name' and 'description' fields in link data. For your video ad, use 'title' and 'description' fields in video data.
from /usr/local/lib/ruby/gems/2.3.0/gems/facebook_ads-0.1.12/lib/facebook_ads/base.rb:159:in `exception'
```

Using the v2.9 API because without it the gem doesn't seem to work.

**Comment by icole** [comment]
Fixed over at PR #18

**Comment by cte** [comment]
Merged. Will publish the new version shortly.
